book.rb

```diff
@@ -0,0 +1,14 @@
+require './rental'
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(person, date)
+    @rentals << Rental.new(person, date, self)
+  end
```

Comment:
- [ ] Remember that in the Rental class in lines 9 and 10 the rental adds itself to the arrays of book and person object so we don't need to duplicate that here.

- [ ] We should also ensure we follow the order of the arguments as outlined in the rental constructor: date, book, person

  ```
  def add_rental(person, date)
    Rental.new(date, self, person)
  end
  ```